commons/all/src/main/java/org/infinispan/commons/configuration/attributes/ConfigurationElement.java

```diff
@@ -38,7 +38,7 @@ protected ConfigurationElement(String element, boolean repeated, AttributeSet at
       this.children = children != null ? children : CHILDLESS;
    }
 
-   public String name() {
+   public final String elementName() {
```

Comment:
Any reason for this change? IMO `ConfigurationElement#ElementName` seems  unnecessarily verbose compared to `ConfigurationElement#name`.

Reply:
Unfortunately many builders were overriding `name()` with their interpretation: we want it to mean the declarative name, not some logical value